src/storage.ts

```diff
@@ -231,7 +231,7 @@ export function createStorage<T extends StorageValue>(
     async setItems(items, commonOptions) {
       await runBatch(items, commonOptions, async (batch) => {
         if (batch.driver.setItems) {
-          await asyncCall(
+          return asyncCall(
```

Comment:
fairly certain this was a bug before. you called setItems and then setItem for every file so files were getting uploaded twice. 

noticed this since the newly introduced customids on uploadthing must be unique and we throw if that's not the case and we got duplicate requests when testing the setItems function before i changed the keyword here

Reply:
Nice catch. Do you mind splitting it into a new PR as well? 🙏🏼

Reply:
https://github.com/unjs/unstorage/pull/392